ui/spinner.js

```diff
@@ -284,10 +286,10 @@ return $.widget( "ui.spinner", {
 	_buttonHtml: function() {
 		return "" +
 			"<a>" +
-				"<span>&#9650;</span>" +
+			"<span>&#9650;</span>" +
```

Comment:
This looks like you formatted the file using esformatter - I dealt with this change by manually reverting it. Indenting string concatenation according to DOM structure is a bit difficult to automate... Considering how trivial this string now is, how about we just collapse it to one or two lines?